src/controllers/pets.ts

```diff
@@ -9,6 +8,9 @@ export class PetsController {
     constructor (app : Express){
         this.#router = Router();
         this.#router.get('/pets', this.handleGetPets.bind(this));
+        this.#router.post('/pets', this.handleCreatePet.bind(this));
+        this.#router.put('/pets', this.handleUpdatePet.bind(this));
+        this.#router.delete('/pets', this.handleDeletePet.bind(this));
```

Comment:
Those routes are for admins so it's better if we put it in abaddress like /admin/*